api/utils.go

```diff
@@ -58,29 +55,31 @@ func resolveBlockTag(
 }
 
 func resolveBlockNumber(
-	number rpc.BlockNumber,
+	blockNumber rpc.BlockNumber,
 	blocksDB storage.BlockIndexer,
 ) (uint64, error) {
-	height := number.Int64()
-
 	// if special values (latest) we return latest executed height
 	//
 	// all the special values are:
+	//	EarliestBlockNumber  = BlockNumber(-5)
```

Comment:
Optional, I would suggest to use `-10` instead of `-5`, in case we need to define other BlockNumber.

Reply:
We don't define this special value, this is actually defined from Geth itself. See here: https://github.com/onflow/go-ethereum/blob/master/rpc/types.go#L66 . By using `-10` we would deviate from Geth, and I want to avoid that.